### ISSUE TITLE
Help Building Code from Fork

### ISSUE THREAD

**Issue by jeremysnyder** [Eliav2/react-xarrows#52]
Been doing JS for a long time, but new to Typescript.  I am seeing several errors in VS Code from the master branch (in my fork), and when I try to `npm run build`, I get 7 errors.

My environment:
```
➜ npm --version
7.6.3
➜ node --version
v12.8.0
➜ tsc --version
Version 3.9.7
```

Appreciate any help in building, I am looking to add support to animate the adding of a line, I have it all figured out in SVG animation, just can't get it to build.

**Comment by Eliav2** [comment]
yes i know. is was fixed in dev branch. please fork dev branch and see if this problem persists

**Comment by Eliav2** [comment]
Dev branch merged to master and react-xarrows v1.6.0 published. Please pull the last changes from master and check

**Comment by jeremysnyder** [comment]
It seems to build just fine now that I have rebased on your master.  Thanks for the quick turnaround!
